predicators/approaches/delivery_approach.py

```diff
@@ -0,0 +1,95 @@
+"""An approach that implements a delivery-specific policy.
+
+Example command line:
+    python predicators/main.py --approach delivery_policy --seed 0 \
+        --env pddl_easy_delivery_procedural_tasks
+"""
+
+from typing import Callable, cast
+
+import numpy as np
+
+from predicators.approaches import BaseApproach
+from predicators.envs.pddl_env import _PDDLEnvState
+from predicators.structs import Action, GroundAtom, State, Task
+
+
+class DeliverySpecificApproach(BaseApproach):
+    """Implements a delivery-specific policy."""
+
+    @classmethod
+    def get_name(cls) -> str:
+        return "delivery_policy"
+
+    @property
+    def is_learning_based(self) -> bool:
+        return False
+
+    def _solve(self, task: Task, timeout: int) -> Callable[[State], Action]:
+
+        def _policy(state: State) -> Action:
```

Comment:
add `# pragma: no cover` to pass the unit-tests:

`def _policy(state: State) -> Action` -> `def _policy(state: State) -> Action  # pragma: no cover`